src/validation.h

```diff
@@ -206,19 +206,23 @@ struct PackageMempoolAcceptResult
 };
 
 /**
- * Try to add a transaction to the mempool. This is an internal function and is
- * exposed only for testing. Client code should use ChainstateManager::ProcessTransaction()
+ * Try to add a transaction to the mempool with a specified acceptance time. This is an internal
+ * function and is exposed only for testing. Client code should use ChainstateManager::ProcessTransaction()
  *
- * @param[in]  active_chainstate  Reference to the active chainstate.
+ * @param[in]  chainparams        Tweakable parameters of this instance.
```

Comment:
I don't think this description "Tweakable parameters of this instance." is very illuminating. Tweakable suggests to me that the user can modify these parameters, but that isn't the case here.

Reply:
Changed the description in 034e827 .